ansible_collections/arista/avd/roles/eos_designs/doc/custom-structured-configuration.md

```diff
@@ -191,3 +190,157 @@ name_server:
   - 10.10.10.11
   - 10.10.10.12
 ```
+## Variables Precedence
+
+_**TL;DR:** eos_cli_config_gen variables in the input files will be overwritten
+by the eos_designs generated facts when the role produces a value for the same
+key, use custom_strutcured_configuration to make sure the values make it in the intended configurations._
+
+### Overview
+
+AVD `eos_designs` role flow is expected to work as follow:
+
+1. from the input variables (inventory, group_vars, host_vars,...), a task first sets ansible facts for each device
```

Comment:
For this description I think the flow could be simplified to:

1. from the input variables (inventory, group_vars, host_vars, ...) the `eos_designs` role generates the "structured_config"
2. The "structured_config" is used as input to the `eos_cli_config_gen` role to generate the intended EOS configuration.

Reply:
addressed in [215e84d](https://github.com/aristanetworks/ansible-avd/pull/1743/commits/215e84d71a018127cdd37073dde4c374dc962906)